base/ltexpl.dtx

```diff
@@ -387,4 +387,78 @@
 %    \end{macrocode}
 %
 %
+%
+%
+%
+%
+% \section{Document-level command names for \pkg{expl3} functions}
+%
+%    Current home for L3 programing layer functions that we make
+%    directly available at the document level. This section may need
+%    to be moved later (after \cs{NewDocumentCommand} is defined in
+%    case we want to use that in the setup.
+%
+%
+% \DescribeMacro\fpval
```

Comment:
Four typos in \DescribeMacro h lines 410-12 also):
  \…val should be \…eval

Reply:
And in my typing!  h  ==>  (

Reply:
good catch, thanks